tick/optim/model/src/variants/hawkes_fixed_kern_loglik_list.h

```diff
@@ -26,6 +26,13 @@ class ModelHawkesFixedKernLogLikList : public ModelHawkesList {
    */
   explicit ModelHawkesFixedKernLogLikList(const int max_n_threads = 1);
 
+  /**
+   * These lines are required for visual studio but are suggested in general
+   *  for a class that should never be copied.
+   */
+  ModelHawkesFixedKernLogLikList(const ModelHawkesFixedKernLogLikList& model) = delete;
```

Comment:
Maybe add that this is necessary to create unique_ptr of this class?

Reply:
sure - it's to stop a copy of the class which contains a vector of unique_ptr